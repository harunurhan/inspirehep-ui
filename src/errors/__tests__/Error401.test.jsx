import React from 'react';
import { shallow } from 'enzyme';
import Error401 from '../components/Error401';

describe('Error401', () => {
  it('renders Error401 with correct props', () => {
    const wrapper = shallow(<Error401 />);
    expect(wrapper).toMatchSnapshot();
  });
});
