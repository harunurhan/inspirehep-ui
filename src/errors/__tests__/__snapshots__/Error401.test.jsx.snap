// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error401 renders Error401 with correct props 1`] = `
<ErrorPage
  imageSrc="401.svg"
  message="Sorry, you are not authorised to view this page."
  statusCode={401}
/>
`;
